src/backend/vulkan/src/command.rs

```diff
@@ -206,7 +206,7 @@ impl command::RawCommandBuffer<Backend> for CommandBuffer {
                         region.src_subresource,
                         region.num_layers,
                     ),
-                    src_offset: offset,
+                    src_offset: offset.clone(),
```

Comment:
I don't mind `Clone` either. `Copy` would work better if we need arrays of that stuff